lib/viewport.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flame/components.dart';
```

Comment:
I personally would like to have the name of the class reflected in the file name.